http/1.6/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -1,35 +1,91 @@
-
-
-== Add Custom Parameters
-
-In addition to the body of the request, you can configure:
+By default, the operation sends the Mule message payload as the HTTP request body but you can customize it using a DataWeave script or expression. In addition to the body of the request, you can configure:
 
 * Request and response size limits
 * Headers
 * Query parameters
 * URI parameters
 * A map of multiple headers or parameters
 
-=== HTTP Request and Response Size Limits
+== Configure HTTP Request Connection
+
+To use an HTTP *Request* operation, the required minimum setting for the operation is a host URI, which can include a path. You can configure a connection with these settings:
+
+* *Host* +
+Indicates where the requests are sent.
+* *Port* +
+Indicates where the requests are received.
+* *Protocol* +
+Indicates the protocol for communication, either HTTP for plain connections or HTTPS for TLS secure connections.
+
+To enable secure connections through HTTPS, you must define a TLS configuration context in the connection and provide:
+
+* A keystore for the server
+* A truststore if you need two-way authentication
+
+To learn about TLS configuration, refer to the xref:mule-runtime::tls-configuration.adoc[Configure TLS with Keystores and Truststores] documentation.
+
+Additionally, you can configure the *Authentication* field with the following types:
+
+* *Basic authentication*
+* *Digest authentication*
+* *Ntlm authentication* (NT LAN Manager)
+
+To learn about authentication types and examples, refer to the xref:http-authentication.adoc[Configure Authentication for HTTP Requests] documentation.
+
+The following example shows how to configure the HTTP *Request* connection in Studio:
+
+. In the *Mule Palette* view, select *HTTP > Request*.
+. Drag *Request* to the Studio canvas.
+. Click the plus sign (*+*) next to the *Connector configuration* field to configure a global element that can be used by all instances of HTTP *Request* in the app.
+. Set *Host* to `localhost` and *Port* to `8081`.
 
-By default, Mule defines different limits for the HTTP request size, response size, and request/response header size. However, you can change these max values by modifying the corresponding property in the `wrapper.conf` file.
+image::http-request-1.png[HTTP Request global configuration window with Host field set to localhost and Port field to 8081]
+[start=5]
+. Set *TLS configuration* to *Edit inline*.
+. In the *Trust Store Configuration* section, set *Path* to `keystore.jks` and *Password* to `Mypassword`.
+
+image::http-request-2.png[Trust Store Configuration window with Path field set to keystore.jks and Password field to Mypassword]
+
+[start=7]
+. Set *Authentication* to *Basic authentication*.
+. Set *Username* to `testuser` and *Password* to `Mypassword`.
+. Click *OK*.
+. In the HTTP *Request* configuration screen, set *Path* to `/testpath`.
```

Comment:
ditto re. colon, GLOBAL.
I will stop asking. Please search for images and evaluate throughout.